conf/layer.conf

```diff
@@ -28,5 +28,4 @@ PREFERRED_VERSION_wpebackend-rdk = "git%"
 PREFERRED_VERSION_icu = "65.%"
 PREFERRED_PROVIDER_virtual/wpebackend = "wpebackend-rdk"
 
-# enable wayland-egl for rpi
-PACKAGECONFIG_append_pn-userland += " wayland"
+include raspberrypi_generic.conf
```

Comment:
Can we have that include in the image files in the recipes-core/images instead of adding that to the global layer conf? I believe layer.conf should contain only metadata for the layer. This would mean that we can also move PREFERRED* to the image files

Reply:
Moved to images